app/src/main/res/values/strings.xml

```diff
@@ -126,4 +126,7 @@
     <string name="title_choose_audio_file">Choose audio file</string>
     <string name="title_del_comment">Delete</string>
     <string name="title_edit_comment">Edit</string>
+    <string name="action_cancel">Cancel</string>
+    <string name="msg_delete_comment">Delete Comment </string>
+    <string name="title_confirm_delete_comment">Are you sure to delete this comment ?</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_confirm_delete_comment` appears to be unused
 Unused resources make applications larger and slow down builds.